package.json

```
@@ -83,24 +83,25 @@
     "babel-polyfill": "^6.9.1",
     "chroma-js": "^1.2.1",
     "classnames": "^2.2.5",
+    "create-react-class": "^15.7.0",
     "d3": "^4.2.1",
     "gulp-uglify-es": "^1.0.4",
+    "history": "^4.10.1",
     "isomorphic-fetch": "^2.2.1",
-    "lodash": "^4.17.11",
+    "lodash": "^4.17.20",
     "mapbox-gl": "^0.50.0",
     "moment": "^2.14.1",
     "openaq-data-format": "github:openaq/openaq-data-format#master",
     "openaq-design-system": "github:openaq/openaq-design-system#feature/v3.0.0",
```

```suggestion
    "openaq-design-system": "github:openaq/openaq-design-system#feature/v3.1.0",
```

In order to get this to work I changed this version number, and then I removed `package-lock.json`, ran `npm install` and then had to manually install `npm install https://github.com/openaq/openaq-design-system\#feature/v3.1.0`. 

@AliceR thoughts? should my modified package-lock be committed now?

---

I forgot to commit this change, thanks! Added with 881f4a7